components/slider/package.json

```
@@ -18,11 +18,13 @@
     "build": "gulp"
   },
   "peerDependencies": {
-    "@spectrum-css/vars": "^8.0.0"
+    "@spectrum-css/stepper": "^4.0.1",
+    "@spectrum-css/tokens": "^7.6.0"
```

```suggestion
    "@spectrum-css/stepper": "^4.0.0",
    "@spectrum-css/tokens": "^7.0.0"
```

As much as possible I'd like to keep peerDependencies at their lowest compatible release.